### PR TITLE
Add lit-analyzer

### PR DESCRIPTION
**Pull request by devpow112**
Found a few minor errors.

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "compilerOptions": {
+    "plugins": [
+      {
+        "name": "ts-lit-plugin",
+        "strict": true,
+        "format": "list",
+        "rules": {
+          "no-unknown-tag-name": "off",
+          "no-complex-attribute-binding": "warn"
```
Comment (devpow112) [inline tsconfig.json:10]: Might just switch it to a property over an attribute eventually. For now just make the `error` a `warn`.
https://github.com/Brightspace/d2l-rubric/blob/master/src/components/icons/rubric-competencies-icon.js#L10
https://github.com/Brightspace/d2l-rubric/blob/master/src/components/rubric-overall-score.js#L279

File: src/components/editor/rubric-autosaving-input.js [hunk 2]
```diff
@@ -32,7 +32,7 @@ class RubricAutoSavingInput extends composeMixins(
 				@focus="${this._onFocus}"
 				@blur="${this._onBlur}"
 				@input="${this._onInput}"
-				aria-invalid="${Boolean(this.invalid).toString()}"
+				aria-invalid="${this.invalid}"
```
Comment (MitchelPaulin) [inline src/components/editor/rubric-autosaving-input.js:35]: Doesn't aria-invalid expect a string? A common pattern I have seen is `aria-invalid="${this.invalid ? 'true : 'false'}`

Comment (devpow112) [reply src/components/editor/rubric-autosaving-input.js:35]: This should serialize to `aria-invalid="true"` or `aria-invalid="false"` but I can make it explicit with the ternary.

Comment (devpow112) [reply src/components/editor/rubric-autosaving-input.js]: Done in 68c4c9c.

**Review by MitchelPaulin — APPROVED** [review]

**Pull request merged**